package.json

```diff
@@ -109,13 +109,17 @@
   },
   "resolutions-netmask-comment": "transitive dep from proxy-agent 4.0.1 & pac-resolver 4.1.0, which are transitive deps from CDK. Remove the forced resolution when able.",
   "resolutions-hosted-git-info": "transitive dep from conventional-changelog-cli@2.1.1 and lerna@4.0.0. hosted-git-info v3.0.0 - v3.0.7 contain a security vulnerability. It is safe to remove/update overrides as long as we stay out of the vulnerable window",
+  "resolutions-meow": "transitive dep from lerna@4.0.0 and standard-version@9.3.0. trim-newlines < 3.0.1 has security vulnerabilities and meow < 10.0.0 uses the vulnerable trim-newlines.",
```

Comment:
this looks like perhaps a copy/paste artifact from #478 ? You probably meant to instead provide a comment for `normalize-url`, right?

Reply:
Oh shoot, thanks for catching that. I actually meant to omit this entirely, I think the one comment about `parse-url` should suffice since it mentions that `normalize-url` is vulnerable for versions < 4.5.1. Removed this line.

Reply:
makes sense - resolved